models/basic-home-example/ISensor.json

```diff
@@ -5,10 +5,9 @@
   "displayName": "Sensor",
   "contents": [
     {
-      "@type": ["Property", "Temperature"],
-      "name": "temperature",
-      "unit": "degreeFahrenheit",
-      "schema": "double"      
+    "@type": "Telemetry",
```

Comment:
Made this a double type because the advanced version shows it as a semantic type, and I'm thinking it would be great to show an example of both.

The Humidity property is a double in this basic example and a semantic type in the advanced example, so I'm following its lead.